ui-v2/tests/steps.js

```diff
@@ -227,7 +227,48 @@ export default function(assert) {
         });
       })
       .then(['I see $property on the $component'], function(property, component) {
-        assert.ok(currentPage[component][property], `Expected to see ${property} on ${component}`);
+        // TODO: Time to work on repetition
+        // Collection
+        var obj;
+        if (typeof currentPage[component].objectAt === 'function') {
+          obj = currentPage[component].objectAt(0);
```

Comment:
So it will only ever select the first object?

Reply:
Yup, if its a collection, sometimes the 'component' isn't a collection, see above, suits me for the moment, definitely will add a 'at position...' or something.